brew/src/main/java/nl/tudelft/serg/evosql/brew/generator/junit/JUnitGenerator.java

```diff
@@ -325,7 +330,7 @@ private MethodSpec generateAfterAll() {
      * @param vendorOptions The vendor options to use.
      * @return A method specification for a single unit test.
      */
-    private MethodSpec generatePathTest(Path path, VendorOptions vendorOptions) {
+    private MethodSpec generatePathTest(Path path, VendorOptions vendorOptions, int expected) {
```

Comment:
This additional parameter is not needed. You're invoking this method with `path.getProductionOutput().size()` as the value of `expected`, while this method is able to simply call this size function itself.

Reply:
Ah that's right, will do